src/osd-overlay/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "react-app"
+  ]
+}
```

Comment:
`"plugin:react/all"` in the base `.eslintrc` has a bunch of rules that conflict with Typescript.